openapi/index_openapi.json

```diff
@@ -1081,10 +1081,21 @@
             "description": "JSON API links object, pointing to a homepage URL for this implementation"
           },
           "link_type": {
-            "$ref": "#/components/schemas/LinkType"
+            "allOf": [
```

Comment:
To me the change looks like an unnecessary complication of schema. AFAIK, the construction before and the one after does the same, but readability became slightly worse. Despite this grunt I am OK with the change :grin:

Reply:
Indeed. This complication is why we originally held back on the update, but as the change is content-wise the same, I don't see the reason why we should keep ourselves held back in `pydantic` updates simply due to this :)

Reply:
For sure. As said, I am fine with it.